### ISSUE TITLE
Question about simple experiment in document

### ISSUE THREAD

**Issue by bconfiden2** [lmcinnes/pynndescent#167]
In the document [How PyNNDescent works](https://pynndescent.readthedocs.io/en/latest/how_pynndescent_works.html), I have a question in **Random projection trees for initialization**.

In the third paragraph of **Random projection trees for initialization**, it says that only one tree is used to initialize the graph and then proceeded with nn-descent for simple testing.
>>>Let’s got back to our example data we built the graph for in the last section. This time we’ll use just one random projection tree to initialize things, and then proceed with nearest neighbor descent as before.

<br>

In my opinion, I don't think nn-descent will work normally in this case.
Because all neighbor's neighbors in node V can only exist within the same leaf block - the k-nearest neighbors of V are already decided, and there is nothing to update with nn-descent.
In other words, if I use only one random projection tree, the node will get stuck in it's leaf block when nn expansioning.

<br>

However, the result shows that nn-descent worked well even though only one random projection tree was used.
I wonder if I misunderstood the concept of random projection tree, or if you used more than two trees to test it in the document.
